lib/util.js

```diff
@@ -29,6 +29,20 @@ function getUserId(userId) {
   return userId
 }
 
+function getUserProperties(userPropertiesAsString) {
+  let result = {}
+  if (!userPropertiesAsString) {
+    return result
+  }
+
+  try {
+    result = JSON.parse(userPropertiesAsString)
+  } catch (error) {
+    // TODO: handle error?
```

Comment:
good question. no logger is there here.
return null.

Reply:
I don't like returning `null`, I prefer `{}` as 0-value